src/util/calendar-helper.ts

```diff
@@ -176,6 +176,34 @@ export namespace CalendarHelper {
             "year"
         );
     };
+
+    export const getWeekStartEnd = (day: Dayjs) => {
+        const firstDayOfWeek = day.startOf("week").format("YYYY-MM-DD");
+        const lastDayOfWeek = day.endOf("week").format("YYYY-MM-DD");
+
+        return {
+            start: firstDayOfWeek,
+            end: lastDayOfWeek,
+        };
+    };
+
+    export const isDisabledDay = (
```

Comment:
this was originally in `calendar-day-style-helper.ts`, moved it here as other variants will likely need this